### PR TITLE
expression: implement vectorized evaluation for 'builtinNowWithoutArgSig'

### PR DESCRIPTION
**Pull request by lwxwx**
### What problem does this PR solve? 
expression: implement vectorized evaluation for 'builtinNowWithoutArgSig'

### What is changed and how it works?
```
goos: windows
goarch: amd64
pkg: github.com/pingcap/tidb/expression
BenchmarkVectorizedBuiltinTimeFuncGenerated-4           1000000000               0.0250 ns/op          0 B/op          0
 allocs/op
BenchmarkVectorizedBuiltinTimeFunc-4                    1000000000               0.0410 ns/op          0 B/op          0
 allocs/op
PASS
```

### Check List 
 - Unit test

### CODE REVIEW

**Comment by francis0407** [comment]
Hi, the CI service is not available now. Please wait a minute.

**Comment by lwxwx** [comment]
test again

File: expression/builtin_time_vec_test.go [hunk 1]
```diff
@@ -69,7 +69,9 @@ var vecBuiltinTimeCases = map[string][]vecExprBenchCase{
 		{retEvalType: types.ETInt, childrenTypes: []types.EvalType{types.ETDatetime}},
 	},
 	ast.MicroSecond: {},
-	ast.Now:         {},
+	ast.Now: {
+		{retEvalType: types.ETInt, childrenTypes: []types.EvalType{types.ETDuration}, geners: []dataGenerator{&rangeDurationGener{0.2}}},
```
Comment (wshwsh12) [inline expression/builtin_time_vec_test.go]: I think the retEvalType should be types.ETDatetime and the childrenTypes should be empty. Isn't it?

Comment (lwxwx) [reply expression/builtin_time_vec_test.go]: Yes, I was wrong.thank you

File: expression/builtin_time_vec.go [hunk 2]
```diff
@@ -896,11 +896,25 @@ func (b *builtinSecondSig) vecEvalInt(input *chunk.Chunk, result *chunk.Column)
...
 
 func (b *builtinNowWithoutArgSig) vectorized() bool {
-	return false
+	return true
 }
 
 func (b *builtinNowWithoutArgSig) vecEvalTime(input *chunk.Chunk, result *chunk.Column) error {
-	return errors.Errorf("not implemented")
+	n := input.NumRows()
+	Now, isNull, err := evalNowWithFsp(b.ctx, int8(0))
```
Comment (b41sh) [inline expression/builtin_time_vec.go]: Variable name should start in lowercase

**Comment by b41sh** [comment]
LGTM

File: expression/builtin_time_vec_test.go [hunk 3]
```diff
@@ -69,7 +69,9 @@ var vecBuiltinTimeCases = map[string][]vecExprBenchCase{
 		{retEvalType: types.ETInt, childrenTypes: []types.EvalType{types.ETDatetime}},
 	},
 	ast.MicroSecond: {},
-	ast.Now:         {},
+	ast.Now: {
+		{retEvalType: types.ETDatetime, geners: []dataGenerator{&rangeDurationGener{0.2}}},
```
Comment (wshwsh12) [inline expression/builtin_time_vec_test.go]: There is no childrenTypes, so the geners is also useless.

**Comment by ekalinin** [comment]
For better transparency it would be better to link current PR with the source issue #12101.
Otherwise other developers do not know that someone is working on the same task.

**Pull request merged**